milestone-2.md

```diff
@@ -0,0 +1,25 @@
+#### Milestone 2
+
+<b>Audience:</b> Mainstream Consumers.
+<b>Overview:</b> Your self-described luddite friend finds Peerlio from a web search, after a brief chat with the Peerfolio Agent, Friend is convinced that they will use Peerfolio as their first venture into acquiring crypto.
+
+
+#### Major enhancements:
+- Product Design for multiplayer mode
+- Technical design support for mainstream consumers - see below
+- [Auto execute conditional swap goals given market conditions](https://github.com/beneviolabs/ft-allowance-agent/issues/26)
+- [Support ETH, SOL, XRP, and BTC in the deposit to token aquisition flow](https://github.com/beneviolabs/ft-allowance-agent/issues/74)
+- Adds [withdraw all / close your trading account](https://github.com/beneviolabs/ft-allowance-agent/issues/75)
+- Adds [SMS Notifications] (https://github.com/beneviolabs/ft-allowance-agent/issues/73)
```

Comment:
All clear 👍. Probably would place the notification registration as part of their set up a future/auto-swap flow.

- What was the reason for choosing SMS notifications? 
- Are users more or less comfortable giving their phone vs email? Personally I find my email easier to give because it is easier to control for spam there.
- Do we need their phone for any other use cases besides notifications? (like KYC)